src/components/AddCustomerForm.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react";
+import "./AddCustomerForm.css";
+
+const AddCustomerForm = props => {
+  const initUser = {
+    id: null,
+    title: "",
+    firstName: "",
+    surname: "",
+    email: "",
+    roomId: " ",
+    checkInDate: "",
+    checkOutDate: ""
+  };
+
+  const [customer, setCustomer] = useState(initUser);
+
+  const handleChange = event => {
+    const updateCustomer = {
+      ...customer,
+      [event.target.name]: event.target.value
+    };
+    setCustomer(updateCustomer);
+  };
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    console.log(customer.email);
+    if (
+      customer.email &&
```

Comment:
Excellent piece of validation to check if the user has filled in all the fields of the form 😄